recipe/meta.yaml

```diff
@@ -20,15 +20,16 @@ requirements:
     - vs2017_{{ target_platform }}    # [win]
   host:
     - python
+    - cffi >=1.12
+    - openssl
```

Comment:
Does the build fail if openssl is not included as a `host` dependency? In the previous release, we just used the rust component.

Reply:
I've just rerun without `openssl`.

Reply:
Yeah; the question here is whether the `cryptography` build process actually requires an external OpenSSL (in which case, explicitly including it in the `host` and `run` is arguably the better thing to do), or whether it just uses a pre-packaged OpenSSL that's pulled in by Rust. 

OpenSSL will always be in the run environment because it is also a Python dependency.

Reply:
Let me know how to do better